nc2pt/conf/config.yaml

```diff
@@ -150,6 +150,7 @@ select:
     # use this to select which years to reserve for testing
     # the remaining years in full will be used for training
```

Comment:
Potentially update comments here now that validation is added?

Reply:
Thanks, good point! Added below